app/vep/vep_resources.py

```diff
@@ -23,29 +23,44 @@
 
 from core.logging import InterceptHandler
 from vep.models.upload_vcf_files import Streamer, MaxBodySizeException
+from vep.models.pipeline_model import *
+from vep.utils.nextflow import launch_workflow
+import json
 
 logging.getLogger().handlers = [InterceptHandler()]
 
 router = APIRouter()
 
 
 @router.post("/submissions", name="submit_vep")
-async def submit_vep(request : Request):
+async def submit_vep(request: Request):
     try:
         stream_obj = Streamer(request=request)
         stream_result = await stream_obj.stream()
-        print(stream_obj.parameters.value.decode())
-        print(stream_obj.genome_id.value.decode())
+        parameters = stream_obj.parameters.value.decode()
+        genome_id = stream_obj.genome_id.value.decode()
+
+        parameters_dict = json.loads(parameters)
+        ini_parameters = ConfigIniParams(**parameters_dict)
+        inifile = ini_parameters.create_config_ini_file(stream_obj.temp_dir)
+        vep_config_parameters = VEPConfigParams(
+            vcf=stream_obj.filepath, vep_config=inifile.name
+        )
+        launch_params = LaunchParams(paramsText=vep_config_parameters, labelIds=[])
+        pipeline_params = PipelineParams(launch=launch_params)
         if stream_result:
-            return {"message": f"Successfully uploaded{stream_obj.filepath}"}
+            workflow_id = launch_workflow(pipeline_params)
+            return {"submission_id": workflow_id}
         else:
             raise Exception
     except MaxBodySizeException:
-        return HTTPException(status_code=status.HTTP_413_REQUEST_ENTITY_TOO_LARGE,
-                             detail='Maximum file size limit exceeded.')
+        return HTTPException(
+            status_code=status.HTTP_413_REQUEST_ENTITY_TOO_LARGE,
+            detail="Maximum file size limit exceeded.",
+        )
     except Exception as e:
         print(e)
-        return HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
-                            detail='There was an error uploading the file.')
-
-
+        return HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="There was an error uploading the file.",
```

Comment:
This message should be updated since it also catches other errors (e.g. thrown when creating ini file or submitting workflow).